python/empyrical_evidences/README.md

```diff
@@ -0,0 +1,106 @@
+<h1 align="center">Empyrical Evidences</h1>
+
+<p align="center">
+  Search Hacker News for comments relevant to your favorite academic papers.
+  Powered by Together.ai and DBOS.
+</p>
+
+## What It Does
+
+This application searches Hacker News for comments relevant to your favorite academic papers.
+Specifically, it uses the Together.ai inference APIs to identify key topics in papers, then searches Hacker News for related comments and ranks them.
+It is serverlessly hosted on DBOS Cloud.
+
+## How It Works
+
+- First, you upload an academic paper. The app uses the Together.ai API to query the paper's embeddings and store them in Postgres using pgvector.
+- Then, you search Hacker News for comments relevant to the paper.
+  The app does this with a multi-agent workflow that first extracts the paper's key topics, then searches Hacker News for relevant comments, then ranks the comments.
+
+## Why Use DBOS
+
+### Serverless
+
+- Go from local to production cloud deployment in 30 seconds with a single command.
+- [15x cheaper](https://www.dbos.dev/blog/dbos-vs-lambda-cost), [25x faster](https://www.dbos.dev/blog/dbos-vs-aws-step-functions-benchmark). 0 devops.
+
+### Durable, transactional execution
+
+- Workflows are always resumed where they left off. Save money and time by not querying APIs again for already-executed steps.
+- Workflows steps have exactly-once or at-least-once guarantees.
+
+### Observability
+
+- DBOS Transact applications automatically generate and export OpenTelemetry traces.
```

Comment:
DBOS automatically generates and exports OpenTelemetry traces for your application